### PR TITLE
Adding final license components from John

### PR DESCRIPTION
**Pull request by shwetaoj**

### CODE REVIEW

**Review by ashahba — APPROVED** [review]
Thanks @shwetaoj 
LGTM

File: third-party-programs.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 intel-optimized tensorflow Third Party Programs File
 
-This file is the "third-party-programs.txt" file specified in the associated Intel end user license agreement for the Intel software you are licensing.
+
+This file contains the list of third party software (�third party programs�) contained in the Intel software and their required notices and/or license terms. This third party software, even if included with the distribution of the Intel software, may be governed by separate license terms, including without limitation, third party license terms, other Intel software license terms, and open source software license terms. These separate license terms govern your use of the third party programs.
```
Comment (ashahba) [inline third-party-programs.txt]: `(�third party programs�)` formatting issue

File: third-party-programs.txt [hunk 2]
```diff
@@ -44,10 +45,7 @@ Third party programs and their corresponding required notices and/or license ter
     importlib-metadata
     Copyright 2017-2019 Jason R. Coombs, Barry Warsaw
 
-    Intel(R) Math Kernel Library for Deep Neural Networks
-    Copyright 2016-2019 Intel Corporation. Copyright 2018 YANDEX LLC.
-
-    Intel� Optimization for TensorFlow*(Intel tensorflow)
+    Intel� Optimization for TensorFlow*(Intel tensorflow)
```
Comment (ashahba) [inline third-party-programs.txt]: same here

Comment (wei-v-wang) [inline third-party-programs.txt]: What is IntelAE ? :)

File: third-party-programs.txt [hunk 3]
```diff
@@ -57,7 +55,7 @@ Third party programs and their corresponding required notices and/or license ter
     Copyright 2019 Kenneth Reitz
 
     rsa
-    Copyright 2011 Sybren A. St�vel <sybren@stuvel.eu>
+    Copyright 2011 Sybren A. St�vel <sybren@stuvel.eu>
```
Comment (ashahba) [inline third-party-programs.txt]: same here

**Review by ashahba — COMMENTED** [review]
Some minor test formatting issues.
Other than that, looks pretty good.

**Review by ashahba — CHANGES_REQUESTED** [review]
Besides some minor text formatting issues, looks pretty good.

File: third-party-programs.txt (hunk 2 above)
Comment (ashahba) [reply third-party-programs.txt]: Possibly due to the text editor used

**Review by rsketine — APPROVED** [review]
looks good no differences with cje-tf checkin.

**Review by rsketine — APPROVED** [review]
look good.

File: third-party-programs.txt [hunk 4]
```diff
@@ -1246,55 +1310,55 @@ Mozilla Public License 2.0
...
 Version 2.0
 1. Definitions
-1.1. �Contributor�
+1.1. Contributor
 means each individual or legal entity that creates, contributes to the creation of, or owns Covered Software.
 
-1.2. �Contributor Version�
-means the combination of the Contributions of others (if any) used by a Contributor and that particular Contributor�s Contribution.
+1.2. Contributor Version
+means the combination of the Contributions of others (if any) used by a Contributor and that particular Contributor�s Contribution.
```
Comment (ashahba) [inline third-party-programs.txt]: Text formatting

**Review by ashahba — CHANGES_REQUESTED** [review]
Only one left

**Review by rsketine — APPROVED** [review]

**Review by wei-v-wang — APPROVED** [review]

**Review by ashahba — APPROVED** [review]
LGTM

**Pull request merged**